src/decorator/decoratorConfig.ts

```diff
@@ -8,6 +8,17 @@ const decoratorConfig = (
 ): DecoratorProps => {
   return {
     appname: 'Sykefraværsoppfølging',
+    environment: undefined,
+    onEnhetChanged: (enhet, enhetValue) => {
+      if (enhet) {
+        setEnhet(enhet, enhetValue);
+      }
+    },
+    onFnrChanged(fnr: string | null | undefined): void {},
+    showEnheter: true,
+    showHotkeys: false,
+    showSearchArea: true,
+    urlFormat: undefined,
```

Comment:
Det er lagt til nye properties for v3 konfigurasjonen. Usikker på hvilke vi bør sette her og til hva 🤔 De som er lagt til her er ikke optional.